pkgs/applications/misc/koreader/default.nix

```diff
@@ -37,16 +46,19 @@ in stdenv.mkDerivation rec {
   dontBuild = true;
 
   installPhase = ''
+    # like ln -s but checks that the pointee exists
```

Comment:
Which one doesn't?

Reply:
In the current expression `ln -s "${font-droid}/share/fonts/opentype/NerdFonts/Droid Sans Mono Nerd Font Complete Mono.otf"` makes a broken symlink, and most other font related symlink. In the end I kept none of them, so maybe it's overzealous.

Reply:
We can't fix that with some clever find filter?

Reply:
If you have a better solution I'm all ears.

Reply:
After the update mkLink is used on three static files. Either they exist or they don't. I don't think we need a wrapper for this.

Reply:
I removed the wrapper.